spec/models/order_spec.rb

```diff
@@ -44,8 +54,13 @@
       expect(Order.find(@order_1.id).status).to eq('pending')
       @item_order_1.update(status: 'fulfilled')
       @item_order_2.update(status: 'fulfilled')
+      
       Order.update_order_status_to_packaged
       expect(Order.find(@order_1.id).status).to eq('packaged')
     end
+
+    it 'orders should display in order of status enums on admin dashboard' do
+      expect(Order.order(:status)).to eq([@order_5, @order_1, @order_3, @order_4, @order_2])
```

Comment:
Unless I am missing something, I think this is saying the order expected is : pending, pending(?), packaged, shipped, canceled. 

The story says: packaged, pending, shipped, canceled.

Reply:
I think I am just confused about your commented out tags. The order of the array is correct so long as 0 is packaged and not pending.